config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
interesting config file.. Where are you using this?

Reply:
I don't think I'm actually using it right now, i had just created it incase i wanted to use it in the future? it IS interesting tho.. lol